source/javascripts/all.js

```diff
@@ -1,4 +1,5 @@
 /*
+ *= require _clipboard.min
```

Comment:
Can you move this down to just before the `require _select`, so that `_ga` gets executed first?